arm9/source/main.c

```diff
@@ -49,8 +49,14 @@ static int load_file(const char *filename, uint32_t addr)
 	return 1;
 }
 
+static bool isKtr(void)
+{
+	return (*(int*)SOCINFO & 2) != 0;
+}
+
 int main(int argc, char *argv[])
 {
+	const char* dtb_filename;
```

Comment:
In the current code, all pointers are placed to the right: `const char *dtb_filename;`
I know I'm being quite picky but let's be consistent for now until we decide on a coding style.

Reply:
this is something I have to agree with, I never use pointers to the left and they look plain ugly when it's inconsistently used